frontend/src/components/ResourcesDisplayCard.jsx

```diff
@@ -26,7 +40,61 @@ export default function ResourcesDisplayCard({ handleClose }) {
           aria-label="Close Card"
         />
 
-        <div className={styles.noteCard}> Put Resources Here</div>
+        <div className={resourceStyles.display_card}>
+          <button
+            type="button"
+            className={resourceStyles.closeButton}
+            onClick={handleClose}
+            aria-label="Close"
+          >
+            &times;
+          </button>
+
+          <h2 className={resourceStyles.modalHeading}>Resources</h2>
+          <nav className={resourceStyles.navBar}>
+            {resources.map((resource) => (
+              <p key={resource.id}>
+                <button
+                  type="button"
+                  onClick={() => changeResource(resource.id)}
+                  className={
+                    currentResourceId === resource.id
+                      ? resourceStyles.active
+                      : ""
+                  }
+                >
+                  {resource.displayName}
+                </button>
+              </p>
+            ))}
+          </nav>
+          <div className={resourceStyles.r_content_card}>
+            {currentResource.content.map((item) => {
+              if (item.type === "text") {
+                return (
+                  <p key={`text-${currentResource.id}`}>
+                    {item.items.map((textItem) => (
+                      <p key={`${textItem}-${currentResource.id}`}>
+                        {textItem}
+                      </p>
+                    ))}
+                  </p>
+                );
+              }
+              if (item.type === "image") {
+                return (
+                  <img
+                    className={resourceStyles.resourceImage}
+                    key={`image-${currentResource.id}`}
```

Comment:
nit: a key allows react to infer re-renders, and requires a unique identifier - adding `image` in front is redundant because it is `id` which makes the item unique